lib/core/versions/latest/AnchoredDataSerializer.ts

```diff
@@ -35,61 +34,53 @@ export default class AnchoredDataSerializer {
     }
 
     const decodedNumberOfOperations = Encoder.decodeAsBuffer(splitData[0]);
-    const numberOfOperations = AnchoredDataSerializer.convertFromThreeBytesBuffer(decodedNumberOfOperations);
+    const numberOfOperations = AnchoredDataSerializer.convertFromBytesBuffer(decodedNumberOfOperations);
 
     return {
       anchorFileHash: splitData[1],
       numberOfOperations: numberOfOperations
     };
   }
 
-  private static convertToThreeBytesBuffer (numberOfOperations: number): Buffer {
+  private static convertToBytesBuffer (numberOfOperations: number): Buffer {
+
+    if (!Number.isInteger(numberOfOperations)) {
+      throw new SidetreeError(ErrorCode.AnchoredDataNumberOfOperationsNotInteger, `Number of operations ${numberOfOperations} must be an integer.`);
+    }
 
     if (numberOfOperations < 0) {
-      // We are going to write the number as unint32
       throw new SidetreeError(ErrorCode.AnchoredDataNumberOfOperationsLessThanZero, `Number of operations ${numberOfOperations} must be greater than 0`);
     }
 
-    if (numberOfOperations > this.maxInThreeBytes) {
-      // We are only using 3 bytes to store the number of operations so any number greater than
+    if (numberOfOperations > this.maxUnsignedIntegerValue) {
+      // We are only using 4 bytes to store the number of operations so any number greater than
       // that is not allowed.
       throw new SidetreeError(ErrorCode.AnchoredDataNumberOfOperationsGreaterThanMax,
-                              `Number of operations ${numberOfOperations} must be less than equal to ${this.maxInThreeBytes}`);
+                              `Number of operations ${numberOfOperations} must be less than equal to ${this.maxUnsignedIntegerValue}`);
     }
 
-    // First write the input into a 4 bytes buffer. Big Endian format.
-    // Example
-    //  input: 1000 will be converted to: [0, 0, 3, 232]
+    // First write the input into a 4 bytes buffer. Little Endian format.
     const byteArrayBuffer = Buffer.alloc(4);
-    byteArrayBuffer.writeUInt32BE(numberOfOperations, 0);
+    byteArrayBuffer.writeUInt32LE(numberOfOperations, 0);
 
-    // Remove the first byte (because of Big Endian) and return the rest
-    return byteArrayBuffer.slice(1, 4);
+    return byteArrayBuffer;
   }
 
-  private static convertFromThreeBytesBuffer (threeBytesBuffer: Buffer): number {
+  private static convertFromBytesBuffer (bytesBuffer: Buffer): number {
 
-    // Ensure that the input has 3 bytes
-    if (threeBytesBuffer.length !== 3) {
-      throw new SidetreeError(ErrorCode.AnchoredDataNumberOfOperationsNotThreeBytes,
-                              `Input must have 3 bytes but have ${threeBytesBuffer.length} bytes instead.`);
+    // Ensure that the input has 4 bytes
+    if (bytesBuffer.length !== 4) {
+      throw new SidetreeError(ErrorCode.AnchoredDataNumberOfOperationsNotFourBytes,

```

Comment:
This line is missing in code coverage, would be nice to add a quick one to make the file 100% once and for all.